pkg/admissioncontroller/webhook/auth/seed/graph/eventhandler_shoot.go

```diff
@@ -137,10 +137,16 @@ func (g *graph) handleShootCreateOrUpdate(shoot *gardencorev1beta1.Shoot) {
 	}
 
 	for _, resource := range shoot.Spec.Resources {
-		// only secrets are supported here
-		if resource.ResourceRef.APIVersion == "v1" && resource.ResourceRef.Kind == "Secret" {
-			secretVertex := g.getOrCreateVertex(VertexTypeSecret, shoot.Namespace, resource.ResourceRef.Name)
-			g.addEdge(secretVertex, shootVertex)
+		// only secrets and configMap are supported here
```

Comment:
Please add unit tests in https://github.com/gardener/gardener/blob/8b3fe1899087fae38849d6ffea67e7b4217b2d78/pkg/admissioncontroller/webhook/auth/seed/graph/graph_test.go#L420